app/hotels/src/singleHotel/roomList/__tests__/RoomDescription.test.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import * as React from 'react';
+import { PlaygroundRenderer } from '@kiwicom/react-native-app-playground';
+
+import { RoomDescription } from '../RoomDescription';
+
+describe('RoomDescription', () => {
```

Comment:
Why do you use these describers? Isn't this obvious from the test name?

Reply:
Guess this is my OCD, `it` looks so naked without a describe.

Just how I was taught writing tests really. I can skip them if you think it is better.

Reply:
I like it more without the description because 1) it's less code and 2) I think it's redundant (verbose):

```
 PASS  app/hotels/src/singleHotel/roomList/__tests__/RoomDescription.test.js
  RoomDescription
    ✓ renders (7ms)
```

Failure (not verbose):

```
 FAIL  app/hotels/src/singleHotel/roomList/__tests__/RoomDescription.test.js
  ● RoomDescription › renders

    expect(value).toMatchSnapshot()
```

However, it's definitely not wrong and we don't have a rule for this so I will accept it. I just wanted to start this discussion. Thank you.